backend/app/controllers/books_controller.rb

```diff
@@ -2,6 +2,18 @@
 
 # Books Controller
 class BooksController < ApplicationController
+  before_action :set_book, only: [:show]
+
+  def index
+    books = Book.all
```

Comment:
Maybe paginate the books by default? Makes for better user experience when the number of books balloons.